lib/diff-plugin-outputs.js

```diff
@@ -61,7 +54,7 @@ module.exports = function(args, resolve) {
   const compareArchive = path.join(tempDir, 'compare.zip');
   try {
     childProcess.execSync(
-      `git archive ${args.ref} ${scriptName} ${fixtures.join(' ')} fixtures/defaults.js -o ${compareArchive}`,
+      `git archive ${args.ref} -o ${compareArchive}`,
```

Comment:
Without specifying paths as parameters, the whole repository is downloaded. It think this is the cleanest and easiest (however not the most resource-saving and fastest) solution to mirror an old version of the repository.